kratos/tests/cpp_tests/modified_shape_functions/test_tetrahedra_3d_4_ausas_incised_shape_functions.cpp

```diff
@@ -0,0 +1,602 @@
+                negative_side_weights,
+                GeometryData::GI_GAUSS_1);
+
+            // Call the interface modified shape functions calculator
+            Matrix positive_interface_side_sh_func, negative_interface_side_sh_func;
+            ModifiedShapeFunctions::ShapeFunctionsGradientsType positive_interface_side_sh_func_gradients, negative_interface_side_sh_func_gradients;
+            Vector positive_interface_side_weights, negative_interface_side_weights;
+
+            tetrahedra_ausas_shape_functions.ComputeInterfacePositiveSideShapeFunctionsAndGradientsValues(
+                positive_interface_side_sh_func,
+                positive_interface_side_sh_func_gradients,
+                positive_interface_side_weights,
+                GeometryData::GI_GAUSS_1);
+
+            tetrahedra_ausas_shape_functions.ComputeInterfaceNegativeSideShapeFunctionsAndGradientsValues(
+                negative_interface_side_sh_func,
+                negative_interface_side_sh_func_gradients,
+                negative_interface_side_weights,
+                GeometryData::GI_GAUSS_1);
+
+            // Call the interface outwards normal unit vector calculator
+            std::vector<Vector> positive_side_area_normals, negative_side_area_normals;
+
+            tetrahedra_ausas_shape_functions.ComputePositiveSideInterfaceAreaNormals(
+                positive_side_area_normals,
+                GeometryData::GI_GAUSS_1);
+
+            tetrahedra_ausas_shape_functions.ComputeNegativeSideInterfaceAreaNormals(
+                negative_side_area_normals,
+                GeometryData::GI_GAUSS_1);
+
+            const double tolerance = 1e-10;
+
+            // Check shape functions values
+            KRATOS_CHECK_NEAR(positive_side_sh_func(0,0), 0.125, tolerance);
+            KRATOS_CHECK_NEAR(positive_side_sh_func(0,1), 0.000, tolerance);
+            KRATOS_CHECK_NEAR(positive_side_sh_func(0,2), 0.125, tolerance);
+            KRATOS_CHECK_NEAR(positive_side_sh_func(0,3), 0.750, tolerance);
+
+            KRATOS_CHECK_NEAR(negative_side_sh_func(0,0), 0.125, tolerance);
+            KRATOS_CHECK_NEAR(negative_side_sh_func(0,1), 0.500, tolerance);
+            KRATOS_CHECK_NEAR(negative_side_sh_func(0,2), 0.250, tolerance);
+            KRATOS_CHECK_NEAR(negative_side_sh_func(0,3), 0.125, tolerance);
+
+            KRATOS_CHECK_NEAR(negative_side_sh_func(1,0), 0.375, tolerance);
+            KRATOS_CHECK_NEAR(negative_side_sh_func(1,1), 0.250, tolerance);
+            KRATOS_CHECK_NEAR(negative_side_sh_func(1,2), 0.250, tolerance);
+            KRATOS_CHECK_NEAR(negative_side_sh_func(1,3), 0.125, tolerance);
+
+            KRATOS_CHECK_NEAR(negative_side_sh_func(2,0), 0.125, tolerance);
+            KRATOS_CHECK_NEAR(negative_side_sh_func(2,1), 0.250, tolerance);
+            KRATOS_CHECK_NEAR(negative_side_sh_func(2,2), 0.375, tolerance);
+            KRATOS_CHECK_NEAR(negative_side_sh_func(2,3), 0.250, tolerance);
```

Comment:
The Ausas sh. func. tests were created before having the `KRATOS_CHECK_VECTOR_NEAR` and `KRATOS_CHECK_MATRIX_NEAR`. We should use these at least in the new tests since the use of one macro per value greatly affects the compilation time.

Reply:
Ah, right, I'll change that.